deploy/operator.yaml

```diff
@@ -15,6 +15,8 @@ spec:
     metadata:
       labels:
         name: sriov-network-operator
+      annotations:
```

Comment:
can you add this also in `deployment/sriov-network-operator-chart/templates/operator.yaml`?

https://github.com/kramaranya/sriov-network-operator-1/blob/6343fa158260a7f70dcf264a95e758e9393ba7ce/deployment/sriov-network-operator-chart/templates/operator.yaml#L19

It's about help deployment.